### PR TITLE
BaSP-TG: Admin section created

### PR DESCRIPTION
**Pull request by tobiaspujol**

### CODE REVIEW

**Review by martinalbrnz — APPROVED** [review]
- Auto generated id works correctly
- Params inline with the team
- Every endpoint is correctly commented
- Get methods and filters work fine
- Delete, Update and Create works fine

File: package.json [hunk 1]
```diff
@@ -25,7 +25,8 @@
     "node": "14.x"
   },
   "dependencies": {
-    "express": "^4.18.0"
+    "express": "^4.18.0",
+    "nodemon": "^2.0.16"
```
Comment (AlexisPuchetaRR) [inline package.json:29]: es una dev Dependencies nodemon quitarla

File: src/resources/admins.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  const found = admins.find((admin) => admin.id === Number(req.params.id));
+  if (found) {
+    res.json(admins.filter((admin) => admin.id === Number(req.params.id)));
+  } else {
+    res.status(400).json({ msg: `No admins with the id of ${req.params.id}` });
+  }
+});
+
+// Get admins by firstName
+router.get('/firstName=:firstName', (req, res) => {
```
Comment (AlexisPuchetaRR) [inline src/resources/admins.js:21]: idem

File: src/resources/admins.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  const found = admins.some((admin) => admin.lastName === req.params.lastName);
+  if (found) {
+    res.json(admins.filter((admin) => admin.lastName === req.params.lastName));
+  } else {
+    res.status(400).json({ msg: `No admins with the lastName of ${req.params.lastName}` });
+  }
+});
+
+// Get admins by active status
+router.get('/active=:active', (req, res) => {
```
Comment (AlexisPuchetaRR) [inline src/resources/admins.js:41]: idem

File: src/resources/admins.js [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+        });
+      }
+    });
+  } else {
+    res.status(400).json({ msg: `No admins with the id of ${req.params.id}` });
+  }
+});
+
+// Update admin
+router.put('/id=:id', (req, res) => {
```
Comment (AlexisPuchetaRR) [inline src/resources/admins.js:92]: idem

**Review by AlexisPuchetaRR — APPROVED** [review]
Muy buen laburo Toby, no se porque el .gitignore y vercel.json muestran cambios... quizas hay algo con el CRLF o LF

**Pull request merged**